tools/src/protoc-gen-go-grpc/go.mod

```diff
@@ -5,3 +5,8 @@ go 1.22.2
 require google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0
 
 require google.golang.org/protobuf v1.28.1 // indirect
+
+// Resolve GHSA-8r3f-844c-mc37.
+// This is a temporary fix until the next release of google.golang.org/grpc/cmd/protoc-gen-go-grpc.
```

Comment:
Shouldn't a manual go get fix this ?

Reply:
Yes, once a new `google.golang.org/grpc/cmd/protoc-gen-go-grpc` version is available.